app/components/post_list/post_list_utils.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {isDateLine, isStartOfNewMessages} from 'mattermost-redux/utils/post_list';
+
+export function getLastPostIndex(postIds) {
```

Comment:
I think we should move this functions to `mattermost-redux/utils/post_list`

Reply:
Done.